src/test/java/teammates/test/cases/ui/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -135,16 +135,22 @@ private void testSubmitAction() throws Exception {
         submitPage = loginToStudentFeedbackSubmitPage("Alice", "Open Session");
 
         submitPage.fillResponseTextBox(1, 0, "Test Self Feedback");
+        assertEquals("18", submitPage.getResponseTextBoxLengthLabelValue(1, 0));
         submitPage.selectRecipient(2, 0, "Benny Charles");
         submitPage.fillResponseTextBox(2, 0, "Response to Benny.");
+        assertEquals("18", submitPage.getResponseTextBoxLengthLabelValue(2, 0));
         submitPage.selectRecipient(2, 1, "Drop out");
         submitPage.fillResponseTextBox(2, 1, "Response to student who is going to drop out.");
+        assertEquals("45", submitPage.getResponseTextBoxLengthLabelValue(2, 1));
```

Comment:
There isn't a need to add the check to all the responses, doing this more than once seems a little redundant to me, but it's fine if you want to have them. One change to make is to avoid hardcoding the length of the response. You can use the length method of the string instead. For example,

    
    String responseText = "Test Self Feedback";
    submitPage.fillResponseTextBox(1, 0, responseText);
    assertEquals(String.valueOf(responseText.length()), submitPage.getResponseTextBoxLengthLabelValue(1, 0));